### PR TITLE
WIP Bokeh plot feature select

### PR DESCRIPTION
**Pull request by hatsy**
## Overview
- Added feature selection to plots
- Add class labels to plots

## Status
- Overall behavior implemented
- Code refactoring needed

### CODE REVIEW

**Comment by bnaul** [comment]
@hatsy finally had a chance to take a look at this; we did a big refactor (moving a bunch of generic logic into a `baselayer` directory that will be the foundation for new apps) so I fixed the conflicts here and pushed the rebased version to your branch. Couple of things:

1) Plotting test needs to be updated (doesn't currently know how to click the button and submit the feature names).
2) Why did `/plot_features/` change from a GET to a POST? Seems like a GET makes more sense here since we're requesting a plot...

**Comment by bnaul** [comment]
@stefanv @acrellin thoughts on the UI here? feature plot

File: static/js/actions.js [hunk 1]
```diff
@@ -62,7 +65,7 @@ String.prototype.format = function (...args) {
 };
 
 
-// Receive list of projects
+// Receive list of
```
Comment (bnaul) [inline static/js/actions.js:68]: Revert

File: static/js/reducers.js [hunk 2]
```diff
@@ -171,18 +171,41 @@ function misc(state={ logoSpinAngle: 0 }, action) {
...
   featuresets,
+  plots,
   features,
   models,
   predictions,
   notifications,
   expander,
   sklearnModels,
   profile,
+  plots,
```
Comment (bnaul) [inline static/js/reducers.js:208]: Remove

**Comment by acrellin** [comment]
@bnaul the UI looks fine. Basic, but fine.

**Pull request closed without merging**